osu.Game.Rulesets.Catch/Objects/Drawables/DrawablePalpableCatchHitObject.cs

```diff
@@ -12,6 +13,17 @@ public abstract class DrawablePalpableCatchHitObject : DrawableCatchHitObject
     {
         public new PalpableCatchHitObject HitObject => (PalpableCatchHitObject)base.HitObject;
 
+        public Bindable<bool> HyperDash { get; } = new Bindable<bool>();
+
+        public Bindable<float> ScaleBindable { get; } = new Bindable<float>(1);
+
+        public readonly Bindable<int> IndexInBeatmap = new Bindable<int>();
```

Comment:
`BindableBool` and `BindableFloat` exist (the latter is actually much preferred over `Bindable<float>` as the latter does not have precision checking built-in).

also, is there a reason why two of these are get-only auto-properties and the third is a readonly field?

Reply:
I don't think those types are preferred here. I think the features (number step snapping, custom ToString etc.) of `BindableX` types are desired in user setting feature (`Bindable` type was originally in `Configuration` namespace, and later used for other purposes, and thus it has number of configuration use-case specific features remained).
> auto-properties
it is just an oversight. I'll make it readonly fields.